README.md

```diff
@@ -1,2 +1,61 @@
-# bookstore-app
+`React` `ES6` `YARN` `LINTERS` `GITFLOW` <br>
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# BOOKSTORE-APP
+
 A library web app to managae your small collection of books, made with HTML, CSS, JS, React.
+
+## Built With
+
+- React
+- ES Modules
+- Linters
+
+## See Project
+
+[Live demo]()
+
+
+## Getting Started
+
+Clone the repo into your local machine.
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+YARN or NPM should be installed.
+
+
+### Install
+
+run `yarn add` or `npm install`to install dependencies
+
+
+### Usage
```

Comment:
- [ ] [OPTIONAL] Your README looks good so far. In addition to what you have done, I would suggest that you mention and explain well the steps to clone and run your project locally. As your project package management is based on `YARN` please mention it and add the necessary command to run it 😉